src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -210,24 +226,58 @@ The tracing integration will create a [transaction]({%- link _documentation/perf
 const activity = ApmIntegrations.Tracing.pushActivity(displayName, {
     data: {},
     op: 'react',
-    description: `<${displayName}>`,
+    description: `${displayName}`,
 });
 
 // Sometime later ...
 // When we pop the activity, the Integration will finish the span and after the timeout finish the transaction and send it to Sentry
-Integrations.ApmIntegrations.popActivity(activity);
+// Keep in mind, as long as you do not pop the activity, the transaction will be kept alive and not sent to Sentry
```

Comment:
We have now two sentences ending "send to Sentry" and missing punctuation.
Maybe the two sentences could be combined together better?

What information are we trying to convey?